src/view/renderer.js

```diff
@@ -407,6 +406,12 @@ export default class Renderer {
 	 */
 	_updateText( viewText, options ) {
 		const domText = this.domConverter.findCorrespondingDomText( viewText );
+
+		// If this is a new text node and it is not in DOM, it will be created and handled in `_updateChildren`.
+		if ( !domText ) {
```

Comment:
@scofalik will you report a ticket for this case and the one you mentioned in https://github.com/ckeditor/ckeditor5-engine/pull/1083#issuecomment-323760233? And of course, please revert changes which we can't do now.